constants/chatbotCommands.py

```diff
@@ -176,10 +176,16 @@ async def alertUser(fro: str, chan: str, message: list[str]) -> str | None:
     if not (targetID := await user_utils.get_id_from_username(target)):
         return "Could not find user."
 
-    if not (targetToken := await osuToken.get_token_by_user_id(targetID)):
+    all_target_user_tokens = await osuToken.get_all_tokens_by_user_id(targetID)
+    if not all_target_user_tokens:
         return "User offline"
 
-    await osuToken.enqueue(targetToken["token_id"], serverPackets.notification(msg))
+    for target_token in all_target_user_tokens:
+        await osuToken.enqueue(
+            target_token["token_id"],
+            serverPackets.notification(msg),
+        )
```

Comment:
I wonder if this is the correct way to do it.. or if it should go to the primary session only (if there's one)

Reply:
I wasn't really sure tbh